py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -68,13 +68,26 @@ def select(
     indices_tensor = ctx.net.add_constant(
         index_value.shape, to_numpy(index_value)
     ).get_output(0)
-    layer = ctx.net.add_gather(input, indices_tensor, dim)
-    out = layer.get_output(0)
+    out = gather(input, indices_tensor, dim)
     if len(out.shape) != 1:
         layer = ctx.net.add_shuffle(out)
     return layer.get_output(0)
```

Comment:
This line (`return layer.get_output(0)`) will fail since `layer` is no longer defined. It should now return `out`, and line 73 above it would need to be changed to have something like `out = layer.get_output(0)`